### ISSUE TITLE
Not define the PAGE_NAME on profile page

### ISSUE THREAD

**Issue by fizi** [e107inc/e107#5264]
### What e107 version are you using?

Latest Github version (just updated)

### Bug description

Not define the PAGE_NAME on profile page.
Appear "commets" on pagename.

### How to reproduce

Go to profile page.

### Expected behavior

Need define the profile page title.

### What browser(s) are you seeing the problem on?

Firefox

### PHP Version

PHP 8.1.10 (localhost)

**Comment by tgtje** [comment]
@fizi  hope it is this? i can not replicate

profile page title  in English  lan_usersettings: 
`if(!defined('PAGE_NAME'))
{
	define("PAGE_NAME", "User Settings");
}`
When using Hungarian lan_usersettings : this is in language pack :

`define("PAGE_NAME", "Felhasználói beállítások");`  so it misses the IF...
that lang file  is 6 years old, the change was made 4 years ago...

**Comment by fizi** [comment]
This in the lan_usersettings.php but the lan_user.php haven't it.

**Comment by tgtje** [comment]
Would like to help, but i do not know what you expect to see ? eg what does PAGE_NAME is UNDEFINED hold.

The image looks like the comment.php  displaying a styled profile.
**the above part is displayed WHEN** comments are enabled ANd _user/options/prefs_ Allow comments in user profile is ENABLED.
when DISabled profile should display > member profile..

**Comment by Jimmi08** [comment]
I am afraid this is related to using 2x tablerender() - one for the user page and, the second for comments. Check your theme and templates for using PAGE_NAME. The same for breadcrumbs.

**Comment by tgtje** [comment]
Adding : this behaviour happens on profile page (user.php)

**Comment by fizi** [comment]
Yes, this happens on profile page. If the comment disabled - the page name appears normally, if the comment enabled - appears "0 comment"
